tox.ini

```diff
@@ -51,7 +51,8 @@ passenv =
   AWS_DEFAULT_REGION
 deps =
   dnspython
-  pytest
+  pytest==3.3.2
```

Comment:
The issue says to pin to 3.2.2, not 3.3.2.

This difference should be resolved.